src/c_api/c_api.cc

```diff
@@ -1363,7 +1363,13 @@ int MXGetVersion(int *out) {
 #if MXNET_USE_TVM_OP
 int MXLoadTVMOp(const char *libpath) {
   API_BEGIN();
-  tvm::runtime::TVMOpModule::Get()->Load(libpath);
+  tvm::runtime::TVMOpModule *libpath_module =  tvm::runtime::TVMOpModule::Get();
+  libpath_module->Load(libpath);
+#if MXNET_USE_CUDA
+  tvm::runtime::TVMOpModule cubin_module;
+  cubin_module.Load("libtvmop.cubin");
```

Comment:
it should combine with the `libpath`, otherwise it is loading from ./libtvmop.cubin

Reply:
Fixed. Thanks!